### PR TITLE
Refactor for integration with planet4 environment

### PR DESCRIPTION
**Pull request by pdewouters**

### CODE REVIEW

File: templates/views/partials/field-tel.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="p4fb__field p4fb__field-{{ field.type | e('html_attr') }} p4fb__field-{{ field.name | e('html_attr') }}">
+    <div class="p4fb__field__element p4fb__field__element--{{ field.type | e('html_attr') }} form-group">
+        <input class="{{ field.class }} p4fb__field__input p4fb__field__input--{{ field.type | e('html_attr') }} form-control"
+               type="tel"
+               pattern="\d{2}-(\d{8}|\d{9})"
```
Comment (mikelittle) [inline templates/views/partials/field-tel.twig:5]: Is this going to work for non-US numbers?

File: templates/views/single-p4-form.twig [hunk 2]
```diff
@@ -1,21 +1,25 @@
-<form id="p4en_form" action="{{ form_submit_url }}" method="post" class="p4fb_form p4en_form">
+{% if (form_errors) %}
+	<div class="errors"><p>There were errors. Please check your answer.</p></div>
+{% endif %}
```
Comment (mikelittle) [inline templates/views/single-p4-form.twig]: I think the error message would be better inside the form tag for accessibility.

**Review by mikelittle — APPROVED** [review]
Thanks for fixing all my issues! 
I have one question and one suggestion. Otherwise, it all looks good.

File: templates/views/partials/field-tel.twig (hunk 1 above)
Comment (pdewouters) [reply templates/views/partials/field-tel.twig:5]: this is the format according to the specs they provided https://www.greenpeace.org.br/human-made-crm-integration

**Review by mikelittle — APPROVED** [review]

**Pull request merged**